src/OpenSage.Game/Data/Apt/AptFile.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using OpenSage.Data.Apt.Characters;
+using OpenSage.Data.Utilities.Extensions;
+
+namespace OpenSage.Data.Apt
+{
+    public class AptFile
+    {
+        public ConstantData Constants { get; private set; }
+        public Movie Movie { get; private set; }
+        internal bool IsEmpty = true;
+        internal string MovieName;
+        internal FileSystem FileSystem;
+
+        public AptFile(ConstantData constants, FileSystem filesystem, string name)
+        {
+            Constants = constants;
+            FileSystem = filesystem;
+            MovieName = name;
+        }
+
+        public void Parse(BinaryReader reader)
```

Comment:
Could this be a private method? I don't think it's called from anywhere other than `AptFile.FromFileSystemEntry`.